### PR TITLE
Updating Readme file.

### PR DESCRIPTION
**Pull request by msjasteppe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,16 @@
...
 
-Device data can be written to directly to the IoMT FHIR Connector for Azure or seamlessly used in concert with other Azure IoT solutions (IoT Hub and IoT Central).  The connector does not provide device security or management which are covered by the Azure IoT solutions mentioned.
+Device data can be written to directly to the IoMT FHIR Connector for Azure or seamlessly used in concert with other Azure IoT solutions (IoT Hub and IoT Central). Please be aware the the connector itself is an open-source project and does not provide device security or management. The Azure IoT solutions themselves support compliance with national, regional, and industry-specific requirements.
+
+The IoMT FHIR Connector for Azure is intended only for use in transferring and formatting data.  It is not intended for use as a medical device or to perform any analysis or any medical function and the performance of the software for such purposes has not been established.  You bear sole responsibility for any use of this software, including incorporation into any product intended for a medical purpose.
```
Comment (kyclai) [inline README.md:9]: > or any medical function and the performance

Add a comma before "and the performance" to break up this long sentence?

File: README.md [hunk 2]
```diff
@@ -4,18 +4,16 @@
 
 The IoMT FHIR Connector for Azure is an open-source project for ingesting data from IoMT (internet of medical things) devices and persisting the data in a FHIR&reg; server. The goal of this Microsoft Healthcare project is to enable developers to rapidly deploy a service for ingesting high frequency IoMT data and landing the data in a FHIR server of their choice.
 
-Device data can be written to directly to the IoMT FHIR Connector for Azure or seamlessly used in concert with other Azure IoT solutions (IoT Hub and IoT Central).  The connector does not provide device security or management which are covered by the Azure IoT solutions mentioned.
+Device data can be written to directly to the IoMT FHIR Connector for Azure or seamlessly used in concert with other Azure IoT solutions (IoT Hub and IoT Central). Please be aware the the connector itself is an open-source project and does not provide device security or management. The Azure IoT solutions themselves support compliance with national, regional, and industry-specific requirements.
```
Comment (kyclai) [inline README.md:7]: > written to directly to

Please change to:
directly written to

**Review by kyclai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (msjasteppe) [reply README.md:9]: Comma added.

File: README.md (hunk 2 above)
Comment (msjasteppe) [reply README.md:7]: Updated as suggested.

**Review by kyclai — APPROVED** [review]

**Pull request merged**